### PR TITLE
Added support for passing _GET variables upon login

### PR DESCRIPTION
**Pull request by MeLlamoPablo**
Hello again. I'm still developing my project wich I've [uploaded to GitHub](https://github.com/MeLlamoPablo/NotaStacks) (in case you wanted to check it out if you were interested. And don't worry, I'll credit you when it's done :P). This time I needed to pass variables to be retrieved after the user logs in, and the framework currently didn't have support for that. So, I've added it, and I've uploaded it to my fork, in case you wanted to consider merging it. Here's how it works.

The ```steamlogin()``` function works as usual. The only change is that you now have to put ```echo``` before calling it: the function itself won't echo the button anymore, you have to manually do it. Why? Felxibility. Sometimes you don't need the button to be ```echo```ed immediatly, but stored it in a variable to maybe pass it as a parameter to a method. For example, in [my script](https://github.com/MeLlamoPablo/NotaStacks/blob/master/menu.php#L27) I pass the button as a parameter to the method ```Modal::__construct()```. The class will create a [Bootstrap Modal](http://getbootstrap.com/javascript/#modals) later, and include the button on it. If ```steamlogin()``` did ```echo``` instead of ```return``` I'd need to modify my entire class; this way is much more convenient.

So, any app that ran
```php
steamlogin();
```
should now run
```php
echo steamlogin();
```
And now, the most important change. If you want to pass variables to be returned after the login, you'd do the following:
```php
echo steamlogin("foo=1&bar=2");
```
Notice that you don't have to put any ```?```s and ```&```s at the beginning.
In that case, you can use ```$_GET['foo']``` and ```$_GET['bar']``` on your login page (set on the settings), and they will be set to ```1``` and ```2```, respectively. Assuming that your return URL is ```http://website.com/index.php```, the user will be redirected to ```http://website.com/index.php?foo=1&bar=2``` after loging in successfully on Steam.

One last change: the line ```session_start()``` at line 37 on the old file was removed because it's already called at the beginning, so I believe that there's no point on calling it again (In fact, it gives you a warning)

Thanks for your time, and have a nice day!

### CODE REVIEW

**Comment by BlackCetha** [comment]
You can actually use `http_build_query(array arguments)` to build your parameter list, so you can just provide an array for `$additional_parameters`.

**Comment by MeLlamoPablo** [comment]
That's true, and more convenient. I didn't think about that. So the way of doing this would be, then:
```php
$vars = http_build_query(array('var1' => 'foo', 'var2' => 'bar'));
echo steamlogin($vars);
```

**Comment by BlackCetha** [comment]
Do that inside your function

**Comment by MeLlamoPablo** [comment]
Okay, I've changed it. I've also updated the example file and the readme with the changes.

**Comment by BlackCetha** [comment]
One last thing, then were done: Its additional, not aditional.
Could you fix that please?

**Comment by MeLlamoPablo** [comment]
Sure thing. English is nor my first language, sorry.

File: README.md [hunk 1]
```diff
@@ -25,27 +25,53 @@ Add your API-Key from http://steamcommunity.com/dev/apikey
 
 Now in your file add the following at the top:
 
-    <?php
+```php
```
Comment (Skaronator) [inline README.md]: typo

Comment (MeLlamoPablo) [inline README.md]: It's not a typo, it's the way of telling GitHub wich sintax will you be using for code highlighting

Comment (Skaronator) [inline README.md]: yeah thats github formatting, thought it was code and not the readme

**Pull request merged**